nomi-labs/assets/nomilabs/lang/zh_cn.lang

```diff
@@ -489,43 +455,95 @@ tooltip.nomilabs.hand_framing_tool.front=前面: %s
+tooltip.nomilabs.tierfiveship.description=从末地微宇宙采集奇异的材料。
+tooltip.nomilabs.tiersixship.description=从每一个可理解的现实中获取极其稀有的材料。
+tooltip.nomilabs.tiersevenship.description=从微宇宙中狩猎龙和末影龙，并从它们的甲壳中携带战利品返回。
+tooltip.nomilabs.tiereightship.description=从末地微宇宙中猎杀无法想象的怪物，并带着奇异的小饰品返回。
+tooltip.nomilabs.tiereightandhalfship.description=从微宇宙之外未经探索的角落收获独特的宝石和金属。
+tooltip.nomilabs.tiernineship.description=坍缩微宇宙恒星并收获它们美味的中子核心。
+tooltip.nomilabs.tiertenship.description1=坍缩整个微宇宙并完全采集它。
+tooltip.nomilabs.tiertenship.description2=如入无人之地。
 
-tooltip.stabilized_miners.description1=A stabilized version, injected with a Heart of a Universe.
-tooltip.stabilized_miners.description2=Lasts indefinitely. Reusable. Totally not overpowered.
-tooltip.stabilized_miners.description3=It looks oddly familiar.
+tooltip.stabilized_miners.description1=一个稳定的版本，注入了宇宙之心。
+tooltip.stabilized_miners.description2=无限期持续。可重复使用。完全不 OP。
+tooltip.stabilized_miners.description3=它看起来奇怪地熟悉。
 
-tooltip.stabilized_matters.description=An infinite and incomprehensible abyss of materials.
+tooltip.stabilized_matters.description=一个无限的、费解的材料深渊。
 
-tooltip.nomilabs.capacitors.description=Ender IO Machine Upgrade
-tooltip.nomilabs.capacitors.level=Level: %s
+tooltip.nomilabs.capacitors.description=Ender IO 机器升级
+tooltip.nomilabs.capacitors.level=级别：%s
 
 # Multiblocks
 tooltip.nomilabs.microverse_projector_1.description=一种能够稳定投影进入微型宇宙中的机器，用于基本的微型采矿任务。
 tooltip.nomilabs.microverse_projector_2.description=一种能够稳定投影进入微型宇宙中的高级机器，用于基本的复杂采矿任务。
 tooltip.nomilabs.microverse_projector_3.description=一种能够稳定投影进入微型宇宙中的杰出机器，用于基本的奇特采矿任务。
-tooltip.nomilabs.creative_tank_provider.description=A complex machine capable of turning buckets and molds into infinite resovoirs beyond human comprehension.
+tooltip.nomilabs.creative_tank_provider.description=一台复杂的机器，能够将水桶和模具变成人类无法理解的无限容器。
 tooltip.nomilabs.naquadah_reactor_1.description=一种高级的反应堆，通过利用富集硅岩和超能硅岩的衰变产生能量。
 tooltip.nomilabs.naquadah_reactor_2.description=一种杰出的反应堆，可从富集硅岩和超能硅岩的衰变中获取更多能量。
 tooltip.nomilabs.naquadah_reactor.produces=正好产生 %s 安培的 %s。
 tooltip.nomilabs.naquadah_reactor.overclock=不要超频!
-tooltip.nomilabs.actualization_chamber.description=A powerful machine that harnesses the most complex of matters and turns it into raw materials.
-tooltip.nomilabs.universal_crystallizer.description=An immense device, capable of turning raw materials into complex matters.
-tooltip.nomilabs.dme_sim_chamber.description=A high powered simulation device, capable of performing more complex simulations faster than ever.
-tooltip.nomilabs.greenhouse.description=A high powered growing chamber, capable of mass-growing even the most incomprehensible plants.
\ No newline at end of file
+tooltip.nomilabs.actualization_chamber.description=一台强大的机器，可以利用最复杂的物质并将其转化为原材料。
+tooltip.nomilabs.universal_crystallizer.description=一种巨大的装置，能够将原材料转化为复杂物质。
+tooltip.nomilabs.dme_sim_chamber.description=一种高功率模拟设备，能够比以往更快地执行更复杂的模拟。
+tooltip.nomilabs.greenhouse.description=一个高功率的生长室，即使是最难以理解的植物也能大规模生长。
+
+
```

Comment:
Yep, all entries below this have been REMOVED, not missing. can you delete them please?

Thank you for doing this though, but just know that you can trust the english langs in this repository, they are auto-generated from the english langs in the original repository.

If you do feel like something might be missing, what you have done here is a great way of bringing that to our attention, but you could also ping me in the translations thread.

Apart from this, its all good.